plugins/zynaddsubfx/CMakeLists.txt

```diff
@@ -153,9 +153,15 @@ SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${PLUGIN_DIR}")
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 ADD_EXECUTABLE(RemoteZynAddSubFx RemoteZynAddSubFx.cpp "${WINRC}")
 INSTALL(TARGETS RemoteZynAddSubFx RUNTIME DESTINATION "${PLUGIN_DIR}")
-SET(FLTK_FILTERED_LIBRARIES ${FLTK_LIBRARIES})
-LIST(REMOVE_ITEM FLTK_FILTERED_LIBRARIES "${X11_X11_LIB}" "${X11_Xext_LIB}")
-TARGET_LINK_LIBRARIES(RemoteZynAddSubFx zynaddsubfx_gui ZynAddSubFxCore ${FLTK_FILTERED_LIBRARIES} -lpthread )
+
+SET(FLTK_FILTERED_LDFLAGS ${FLTK_LIBRARIES})
+FIND_PROGRAM(FLTK_CONFIG fltk-config)
+IF(FLTK_CONFIG AND NOT (LMMS_BUILD_APPLE OR LMMS_BUILD_WIN32))
```

Comment:
@jasp00 Thanks!

Can we get a small comment in the file about why this is required.... e.g. Remove unnecessary X11 linking or whatever the original issue description/pr description was?